keyboards/preonic/keymaps/juno/keymap.c

```diff
@@ -0,0 +1,314 @@
+/* Juno's Preonic Layout  */
+
+#include "preonic.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```